packages/clerk-js/src/ui/elements/CodeControl.tsx

```diff
@@ -178,7 +178,7 @@ export const CodeControl = React.forwardRef<{ reset: any }, CodeControlProps>((p
         gap={2}
         align='center'
       >
-        {values.map((value, index) => (
+        {values.map((value, index: number) => (
```

Comment:
❓ Why do we need the type annotation here?

Reply:
For some reason ESLint/TypeScript didn't infer it correctly. I don't know 🤷

Reply:
I get the correct type locally, but eslint complaints with: 
![image](https://github.com/clerkinc/javascript/assets/1811063/a4833ffc-cdb7-4242-95bf-4bad63589f67)

No blocker in any way so I'm going to resolve this as well.

Reply:
Yeah, I got the same. Rather than type casting that instance of `index` I chose to explicitly type it